dashboards/flutter-android/flutter-android.json

```diff
@@ -0,0 +1,1077 @@
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						},
+						"thresholds": [
+							{
+								"alertSeverity": "CRITICAL",
+								"value": -1
+							}
+						]
+					}
+				},
+				{
+					"title": "App Crashes by Version",
+					"layout": {
+						"column": 5,
+						"row": 1,
+						"width": 2,
+						"height": 3
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.bar"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"nrqlQueries": [
+							{
+								"accountIds": [],
+								"query": "SELECT count(*) FROM MobileCrash FACET appVersion WHERE appName = {{appName}}"
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						}
+					}
+				},
+				{
+					"title": "App Usage by Country",
+					"layout": {
+						"column": 7,
+						"row": 1,
+						"width": 2,
+						"height": 3
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.bar"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"nrqlQueries": [
+							{
+								"accountIds": [],
+								"query": "SELECT uniqueCount(sessionId) as 'App launches' FROM Mobile FACET `countryCode` LIMIT 5 WHERE appName = {{appName}}"
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						}
+					}
+				},
+				{
+					"title": "App Sessions",
+					"layout": {
+						"column": 9,
+						"row": 1,
+						"width": 4,
+						"height": 3
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.billboard"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"nrqlQueries": [
+							{
+								"accountIds": [],
+								"query": "SELECT count(newrelic.timeslice.value) FROM Metric WHERE (metricTimesliceName = 'Session/Start') FACET `instanceName` LIMIT 5 WHERE appName = {{appName}}"
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						}
+					}
+				},
+				{
+					"title": "",
+					"layout": {
+						"column": 1,
+						"row": 2,
+						"width": 2,
+						"height": 2
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.markdown"
+					},
+					"rawConfiguration": {
+						"text": "**About**\n\nInstrument your application with New Relic - [Android Agent](https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-android/get-started/introduction-new-relic-mobile-android/) and\n\n[Flutter Agent](https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-flutter/monitor-your-flutter-application/).\n\n\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=FlutterAndroid) here and let us know how we can improve it for you."
```

Comment:
@rahulbasu-nr need your expertise.

Reply:
Looks good once we remove the space between the 2 lines 'and' & 'Flutter Agent'.

Reply:
I have updated it as per your comments.
Thank you...